css-cascade-5/Overview.bs

```diff
@@ -1035,7 +1035,14 @@ Cascade Sorting Order</h3>
 			For this purpose:
 
 			<ul>
-				<li>Style sheets are ordered as in <a href="https://drafts.csswg.org/cssom/#documentorshadowroot-final-css-style-sheets">final CSS style sheets</a>.
+				<li>Style sheets are ordered as in {insert link to the "final CSS style sheets" concept from CSSOM-1}
+        
+        None of these link styles seem to properly link over to CSSOM:
```

Comment:
Help appreciated here. Bikeshed doesn't like me very much.

Reply:
See, for reference, this conversation: https://github.com/w3c/csswg-drafts/pull/6431#issuecomment-876092520

Reply:
So to solve this: go to [CSSOM terms defined here](https://drafts.csswg.org/cssom/#index-defined-here), then go to [final CSS style sheets](https://drafts.csswg.org/cssom/#documentorshadowroot-final-css-style-sheets) so, I think, you need

```html
[[cssom#documentorshadowroot-final-css-style-sheets]]
```

See https://tabatkins.github.io/bikeshed/#section-links

Reply:
No, it's not a section link, it's a definition:

```
<dfn data-dfn-for="DocumentOrShadowRoot" data-dfn-type="dfn" data-export="" id="documentorshadowroot-final-css-style-sheets">final CSS style sheets</dfn>
```

so `[=final CSS style sheets=]` should work; if it complains, `[=DocumentOrShadowRoot/final CSS style sheets=]` will.